pkg/apis/build/v1alpha2/buildpack_types.go

```diff
@@ -0,0 +1,60 @@
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+
+	corev1alpha1 "github.com/pivotal/kpack/pkg/apis/core/v1alpha1"
+)
+
+const (
+	BuildpackKind   = "Buildpack"
+	BuildpackCRName = "buildpacks.kpack.io"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object,k8s.io/apimachinery/pkg/apis/meta/v1.ObjectMetaAccessor
+
+// +k8s:openapi-gen=true
+type Buildpack struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   BuildpackSpec   `json:"spec"`
+	Status BuildpackStatus `json:"status"`
+}
+
+// +k8s:openapi-gen=true
+type BuildpackSpec struct {
+	// +listType
+	Source             corev1alpha1.StoreImage `json:"source,omitempty"`
```

Comment:
Does this lead to double nesting the image reference?
```
apiVersion: kpack.io/v1alpha3
kind: Buildpack
metadata:
  name: my-custom-app-buildpack-0.3.1
  namespace: my-kpack-using-namespace
spec:
  source:
      image: gcr.io/my-org/my-custom-app-buildpack:0.3.1
  serviceAccountName: my-service-account
```

Reply:
Yep, I tired to keep it similar to ClusterStore's
```yaml
spec:
  sources:
  - image: gcr.io/cf-build-service-public/node-engine-buildpackage@sha256:95ff756f0ef0e026440a8523f4bab02fd8b45dc1a8a3a7ba063cefdba5cb9493
```

And ClusterStack's
```yaml
spec:
  buildImage:
    image: "paketobuildpacks/build:base-cnb"
  runImage:
    image: "paketobuildpacks/run:base-cnb"
```

That said, I'm not attached to it in anyway